addons/website_event_track/controllers/main.py

```diff
@@ -146,11 +148,12 @@ def event_track_proposal_post(self, event, **post):
             'partner_name': post['partner_name'],
             'partner_email': post['email_from'],
             'partner_phone': post['phone'],
-            'partner_biography': escape(post['biography']),
+            'partner_biography': post['biography'],
             'event_id': event.id,
             'tag_ids': [(6, 0, tags)],
             'user_id': False,
-            'description': escape(post['description'])
+            'description': post['description'],
+            'image': base64.b64encode(post['image'].read()) if post.get('image') else False
```

Comment:
We should handle the case when the image content is crap no?

Reply:
Isn't the 'accept' attribute from the input sufficient ? It's currently set to accept all images